doc/deployment/deploy_custom.md

```diff
@@ -11,7 +123,17 @@ This document  details the various options of the `pachctl deploy custom ...` co
 
 ### Preparing your environment
 
-Please see the [introduction to on-premises deployment](./on_premises.html) for steps you need to take prior to creating a Pachydemerm deployment manifest.
+Please see the [introduction to on-premises deployment](./on_premises.html) for steps you need to take prior to creating a custom Pachyderm deployment manifest.
+
+### Customizing through pachctl flags
+
+`pachctl` includes flags for customizing aspects of your deployment,
+from memory and cpu requests for `etcd` and `pachd` to specifying a Kubernetes namespace.
+
+Please check what flags are available in your version of Pachyderm by running `pachctl deploy custom --help`.
```

Comment:
I think this doc should include this information. maybe not every option, but at the very least the ones that are common or most confusing. This is less necessary if we update the --help to be better, but that'd need to be a separate PR.

Reply:
That was in our original plan for these docs: creating a set of PR's to reorganize and clarify those flags. 

I think the most common ones are the cpu and memory requests.  It would be nice to get those instrumented so we can collect data on how people use them...